lxd/patches.go

```diff
@@ -188,6 +190,30 @@ func patchesApply(d *Daemon, stage patchStage) error {
 
 // Patches begin here
 
+func patchDnsmasqEntriesIncludeDeviceName(name string, d *Daemon) error {
+	nodeId := strconv.Itoa(int(d.cluster.GetNodeID()))
+	return d.cluster.InstanceList(&db.InstanceFilter{Node: &nodeId}, func(inst db.Instance, proj db.Project, profiles []api.Profile) error {
+		for _, device := range inst.Devices {
+			if device.Type == db.TypeNIC && device.Config["nictype"] == "bridged" && device.Config["network"] != "" {
```

Comment:
This won't clean up entries that use the parent property rather than network.

Can't we just blow away all the config files and rebuild all nic hosts files to be safe we recreate the correct config?

Reply:
I'm not sure this patch will work as those with network property don't have nictype.

Reply:
Yeah, wiping them all may be the best approach, hopefully dnsmasq won't get too mad when that happens (as the instances will still be running during the refresh).

Reply:
@markylaing can you send a follow up PR to sort that out on Monday?

Reply:
From memory dnsmasq notices adding or updating files but not removing them so should be OK.